src/main.ts

```diff
@@ -461,7 +461,7 @@ export async function validatePipeline(pipeline: string): Promise<boolean> {
     if (axios.isAxiosError(error) && error.response) {
       if (error.response.status === 400) {
         const errorMessage =
-          error.response?.data?.violations.map(violation => violation.message).toString() ||
+          error.response?.data?.violations.map(violation => (violation.message, violation.field)).toString() ||
```

Comment:
Linter is also complaining, but I think here you should append the field only when it exists and adding some prefix like "Error validating field:..."

Reply:
Actually, maybe check with Dani if there is always field and if there is then you can change the message to something like:

`Field: ${violation.field}. Error: ${violation.message}`

Reply:
According to [documentation](http://developers.eng.vmware.com/apis/content-platform/latest/data-structures/ConstraintsViolation/), it looks like there is always `field`. I have checked it with Dani.

Reply:
Done